src/utils/version.js

```diff
@@ -19,6 +19,32 @@ import LOCAL_STORE from '../storage';
  */
 
 const version = {
+
+  /**
+   * Get version data from HathorLib
+   * @returns {Promise} Promise that resolves after getting the version
+   * @memberof Version
+   */
+  async getVersionData() {
```

Comment:
Get version data from api instead of wallet facade to allow calling from the locked screen

Reply:
Yes, but it will only work in the old facade, so this is breaking the wallet service